docs/manual/about/under-the-hood/guarantees.md

```diff
@@ -13,6 +13,52 @@ well as how we label the stability of our components. Next, you can head over to
 the [components page][pages.components] and use filters to see which components
 support specific guarantees.
 
+## End-to-End Acknowledgement Guarantees
+
+<ul class="connected-list">
+<li>
+
+### Enabled
+
+This indicates that acknowledgements are fully supported for the given
+source or sink component.
+
+For a source, this means that the component is both able, at a protocol
+level, to indicate to the sender of the events a status code when and if
```

Comment:
```suggestion
For a source, this means that the component is able at the protocol
level to both indicate to the sender of the events a status code when and if
```

Reply:
I think this changed version isn't quite accurate. The status code indication happens at the protocol level, while the source waiting happens within vector. Re-reading it, the original is poorly worded as well. I'll try again.